java/kafka/deployment-oauth.yaml

```diff
@@ -71,7 +71,7 @@ spec:
                   name: sso-x509-https-secret
                   key: tls.crt
             - name: KAFKA_BOOTSTRAP_SERVERS
-              value: my-cluster-kafka-bootstrap:9093
+              value: my-cluster-kafka-bootstrap:9092
```

Comment:
Same as below.

Reply:
@scholzj I had the port set at 9093 for the reason you outlined in your comment. It was only when I saw an older commit [1] (from before I started) that had set the port to 9092 that I decided to revert to 9092. I can change to 9093 again if preferred.

[1] https://github.com/strimzi/client-examples/blob/3f7a8496e03431ab2bcf9f530178e4f9cf074577/java/kafka/deployment-oauth.yaml

Reply:
In the commit you linked to, the port 9092 was correct because it was not using TLS for connecting to Kafka. But now it is using it. So now it should be 9093.

Reply:
Ah of course. Changing now.